wpcsite/settings.py

```diff
@@ -242,19 +266,31 @@ def rel_path(*subpaths):
         ),
         'APP_DIRS': True,
         'OPTIONS': {
-            'context_processors': [
-                'django.contrib.auth.context_processors.auth',
-                'django.template.context_processors.debug',
-                'django.template.context_processors.i18n',
-                'django.template.context_processors.media',
-                'django.template.context_processors.static',
-                'django.template.context_processors.tz',
-                'django.contrib.messages.context_processors.messages',
-            ]
+            'context_processors': _CONTEXT_PROCESSORS,
         }
     },
 ]
 
+# L10n extraction configuration
+PUENTE = {
+    'BASE_DIR': BASE_DIR,
+    'DOMAIN_METHODS': {
+        'django': [
+            ('**/migrations/**', 'ignore'),
+
+            ('bcauth/**.py', 'python'),
+            ('bcauth/**/jinja2/**.html', 'jinja2'),
+
+            ('mdn/**.py', 'python'),
+            ('mdn/**/jinja2/**.html', 'jinja2'),
+
+            ('webplatformcompat/**.py', 'python'),
+            ('webplatformcompat/**/jinja2/**.html', 'jinja2'),
```

Comment:
The above rules cover the files that get looked at for strings extraction. This doesn't cover django-allauth things.

To be honest, I'm not really sure how to set that up since we don't know where those files will be. That's an intriguing issue. I vote we write up a bug for it and push it off until then.